Saudox/resources/views/profissional/anamnese/psicopedagogia/criar.blade.php

```diff
@@ -1,5 +1,1161 @@
+                            <input type="text" name="imita_algum_som_especifico_se_sim_quais" value="{{ old2($anamnese, "imita_algum_som_especifico_se_sim_quais") }}" />
+
+                            <h3>Mostra-se sonolento? Com qual frequência?</h3>
+                            <input type="text" name="mostrase_sonolento_se_sim_com_qual_frequencia" value="{{ old2($anamnese, "mostrase_sonolento_se_sim_com_qual_frequencia") }}" />
+
+                            <h3>Quando estimulado, consegue relembrar de situações vivenciadas?</h3>
+                            <input type="radio" name="quando_estimulado_consegue_relembrar_de_situacoes_vivenciadas" value="1" {{ old_checked($anamnese, "quando_estimulado_consegue_relembrar_de_situacoes_vivenciadas", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="quando_estimulado_consegue_relembrar_de_situacoes_vivenciadas" value="0" {{ old_checked($anamnese, "quando_estimulado_consegue_relembrar_de_situacoes_vivenciadas", "0") }} /><label>Não</label><br>
+
+                            <h3>Com que frequência ignora estímulos?</h3>
+                            <input type="text" name="com_que_frequencia_ignora_estimulos" value="{{ old2($anamnese, "com_que_frequencia_ignora_estimulos") }}" />
+
+                            <h3>Com que frequência manipula brinquedos e objetos?</h3>
+                            <input type="text" name="com_que_frequencia_manipula_brinquedos_e_objetos" value="{{ old2($anamnese, "com_que_frequencia_manipula_brinquedos_e_objetos") }}" />
+
+                            <h3>Ansiedade em processos de mudança de rotina? Algum exemplo?</h3>
+                            <input type="text" name="ansioso_no_processo_de_mudanca_de_rotina_se_sim_voce_lembra" value="{{ old2($anamnese, "ansioso_no_processo_de_mudanca_de_rotina_se_sim_voce_lembra") }}" />
+
+
+
+
+
+
+                        </div>
+                        <div class="pag_form">
+                            <br>
+                            <br>
+                            <hr class="hr_form">
+                            <h1>Finalização</h1>
+                            <br>
+
+                            <h3>Análise da entrevista</h3>
+                            <textarea class="textareas_form" name="analise_da_entrevista" rows="4" cols="50" style="">{{ old2($anamnese, "analise_da_entrevista") }}</textarea><br><br><br>
+
+                            <h3>Encaminhamentos</h3>
+                            <textarea class="textareas_form" name="encaminhamentos" rows="4" cols="50" style="">{{ old2($anamnese, "encaminhamentos") }}</textarea><br><br><br>
+
+                            <input type="submit" value="Salvar">
+
+
+                        </div>
+
+                        <br>
+                        <div id="container_btn_pags">
+                            <a onclick="form_ante_pag(this)" class="btn_pags redondo">&#8249;</a>
```

Comment:
## Esse 'this', é a página em si?
Se não, o que é? Fiquei na dúvida principalemnte pq na função ele não recebe nada como parâmetro

Reply:
o this é o anchor. é o elemento que tá sendo executado:

`<div onclick="funcao(this)">`   nesse caso é o div. equivalente:

`<div id="id_div" onclick="funcao(document.getElementById('id_div'))">`

Reply:
Entendido!